core-web/apps/dotcms-ui/src/app/view/components/dot-content-compare/components/dot-content-compare-block-editor/dot-content-compare-block-editor.component.html

```diff
@@ -0,0 +1,27 @@
+<div
+    #HtmlWorking
+    *ngIf="label"
+    [innerHTML]="htmlWorkingValue$ | async"
+    data-testId="table-editor-label"
+></div>
+<div
+    #HtmlCompare
+    *ngIf="!label"
+    [innerHTML]="htmlWorkingValue$ | async | dotDiff : htmlCompareValue$ : showDiff"
+    data-testId="table-editor-compare"
+></div>
```

Comment:
See my first example, that way you can just use one async pipe, I think here is creating 2 subscriptions of the same observer